consensus/docker-compose-files/scripts/infiniteloop.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
ah, so the cli container working dir moved to `./fabric/peer`  so the invoke of the shell script becomes `../scripts/infiniteloop.sh`